stdlib/public/Platform/glibc.modulemap.gyb

```diff
@@ -36,6 +36,13 @@ module SwiftGlibc [system] {
   link "execinfo"
 % end
 
+% if CMAKE_SDK in ["FREEBSD"]:
```

Comment:
I wonder why you put `cdefs` outside of `module C`? Is there a particular reason why it's in the global namespace and doesn't belong to a specific module?

Reply:
Yeah, it is not part of the C specification, so its not a submodule of C.  It is a system level module.